src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java

```diff
@@ -204,6 +206,62 @@ public static String buildAppVersionTag(Run<?, ?> build) {
         return build.getParent().getDisplayName() + "-" + build.getNumber();
     }
 
+    /**
+     * Number of traces by severity
+     *
+     * @param traces
+     * @return String with the number of traces for each severity.
+     */
+    public static String getVulnerabilityInfoString(Traces traces) {
```

Comment:
Need lots of tests here. Tests should specify:
* What gets returned if traces is null
* What gets returned if traces is empty
* What gets returned of there are several traces of different severities

Reply:
Tests added